extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -253,6 +264,21 @@ export class LiveListManager {
   }
 }
 
+/**
+ * Detects if a document has had transforms applied
+ * e.g. by a domain with signed exchange domain enabled.
+ * @param {!Document|!ShadowRoot} root
+ * @return {boolean}
+ */
+function isDocTransformed(root) {
```

Comment:
I considered that, but I think it could increase bundle size, and since no other components use this logic yet, that bundle size cost would have no benefits in the short term. Do you agree?

Reply:
don't feel strongly either way. would want to avoid copy/pasting this when next usage comes along, we can deal with it then.